test/__tests__/data/criteo_audience_router_rETL_input.json

```diff
@@ -0,0 +1,124 @@
+[
+  {
+    "destination": {
+      "Config": {
+        "clientId": "abcdef8-f49-4cd6-b4c5-958b3d66d431",
+        "clientSecret": "sjhdkhfrz6yc9LrRRIPimE9h53jADLccXTykHCcA6eEoFR4rXQg",
+        "audienceId": "34893",
+        "audienceType": "email"
+      },
+      "ID": "iwehr83843"
+    },
+    "metadata": {
+      "secret": {
+        "accessToken": "success_access_token"
+      },
+      "jobId": 1
+    },
+    "message": {
+      "userId": "user 1",
+      "type": "audiencelist",
+      "properties": {
+        "listData": {
+          "add": [
+            {
+              "madid": "sample_madid",
+              "email": "alex@email.com",
+              "identityLink": "text.com",
+              "gum": "sdjfds"
+            },
+            {
+              "madid": "sample_madid_1",
+              "email": "amy@email.com",
+              "identityLink": "yahoo.com",
+              "gum": "sdjfds"
+            },
+            {
+              "madid": "sample_madid_2",
+              "email": "van@email.com",
+              "identityLink": "abc.com",
+              "gum": "sdjfds"
+            }
+          ]
+        }
+      },
+      "context": {
+        "externalId": [
+          {
+            "type": "CRITEO_AUDIENCE-23848494844100489",
+            "identifierType": "EMAIL"
+          }
+        ],
+        "mappedToDestination": "true",
+        "sources": {
+          "job_run_id": "cgiiurt8um7k7n5dq480",
+          "task_run_id": "cgiiurt8um7k7n5dq48g",
+          "job_id": "2MUWghI7u85n91dd1qzGyswpZan",
+          "version": "895/merge"
+        }
+      },
+      "timestamp": "2020-02-02T00:23:09.544Z"
+    }
+  },
+  {
```

Comment:
there's no corresponding expected response for this test payload

Reply:
my mistake. forgot to git add the file